dpu-cni/pkgs/cniserver/cniserver.go

```diff
@@ -319,15 +333,8 @@ func NewCNIServer(options ...func(*Server)) *Server {
 	return s
 }
 
-func WithHandler(rqFunc processRequestFunc) func(*Server) {
-	return func(s *Server) {
-		s.processRequest = rqFunc
-	}
-}
-
-func WithSocketPath(runDir string, socketPath string) func(*Server) {
+func WithSocketPath(socketPath string) func(*Server) {
```

Comment:
where is this func used?

Reply:
also should the func be exported?

Reply:
`dpu-cni/pkgs/cniserver/cniserver_test.go`